### ISSUE TITLE
Remove "Predict identifications from trained model" function

### ISSUE THREAD

**Issue by grololo06** [ecotaxa/ecotaxa_dev#722]
From previous conversations, this function is globally unused, "normal" prediction is already a black box to many users and this one is ever darker :)
#142 is also an option there
#151 would disappear
The ugly disk storage of RF models would _not_ have to move to the back-end.

If the goal is to accelerate prediction, we can investigate how.

**Comment by grololo06** [comment]
A single model is there and I did not manage to create another one.
```
root@newecotaxa:/home/ecotaxa/ecotaxa/RF_models# ls -Rls 
.:
total 12
4 drwxrwxrwx 2 www-data www-data 4096 juin  20  2018 Test
4 drwxrwxrwx 2 www-data www-data 4096 nov.  23  2017 TestLN1
4 drwxrwxrwx 2 www-data www-data 4096 avril 13  2018 zooscan_wp2

./Test:
total 1227204
      4 -rw-rw-rw- 1 www-data www-data       3871 juin  20  2018 meta.json
1227200 -rw-rw-rw- 1 www-data www-data 1256646987 juin  20  2018 random_forest.jbl

./TestLN1:
total 173820
     4 -rw-rw-rw- 1 www-data www-data      2964 nov.  23  2017 meta.json
173816 -rw-rw-rw- 1 www-data www-data 177985064 nov.  23  2017 random_forest.jbl

./zooscan_wp2:
total 28098548
       8 -rw-rw-rw- 1 www-data www-data        5845 avril 13  2018 meta.json
28098540 -rw-rw-rw- 1 www-data www-data 28772889464 avril 13  2018 random_forest.jbl

```

**Comment by grololo06** [comment]
The categories are referenced in model's meta.json which means special treatment after WoRMS migration.

**Comment by picheral** [comment]
The idea was to accelerate the prediction... and it did not work because of the size of the model. Unless we find a technical solution, this function will remain unused and can be removed.

**Comment by jiho** [comment]
The real issue is #151 and there is no simple solution to it. So yes, drop for now. If we swap big RandomForest with smaller MLPs with the same performance, we'll revisit.

**Comment by grololo06** [comment]
Fixed in 2.5.13.
